src/tests/ftest/util/dfuse_utils.py

```diff
@@ -235,7 +235,10 @@ def run(self):
         if not self.check_running(fail_on_error=False):
             self.log.info('Waiting five seconds for dfuse to start')
             time.sleep(5)
-            self.check_running()
+            if not self.check_running(fail_on_error=False):
+                self.log.info('Waiting twenty five seconds for dfuse to start')
+                time.sleep(25)
+                self.check_running()
```

Comment:
Wouldn't it be better to loop with a timeout and a smaller interval between checks? This way you're sleeping for 25 seconds regardless of how long it actually takes dfuse to start. In the grand scheme of things maybe it doesn't matter, but when you have thousands of tests these things add up...

Reply:
Yes, it would be better although it would still be polling.

I'm about to start work on https://jira.hpdd.intel.com/browse/DAOS-4465 which would cause dfuse itself to stay in the foreground until it's launched so any delays in daos_init() would block there and this check would then be a near-instant belt-and-braces check, along with errors being reported properly through exit status rather than polling for it being active.